content/interact-with-wallets.md

```diff
@@ -119,16 +120,15 @@ import {
   WalletModalProvider,
   WalletMultiButton,
 } from "@solana/wallet-adapter-react-ui";
-import { PhantomWalletAdapter } from "@solana/wallet-adapter-phantom";
 import * as web3 from "@solana/web3.js";
 
 const Home: NextPage = (props) => {
   const endpoint = web3.clusterApiUrl("devnet");
-  const wallet = new PhantomWalletAdapter();
+  const wallets: Array<WalletAdapter> = [];
```

Comment:
Slightly annoying but probably worth memoizing this so it doesn't change on every render.

Reply:
@mikemaccana can you add this when you have a sec? I'll approve the PR and you can just merge when you're good to go